### ISSUE TITLE
include_role: privilege escalation (nested role)

### ISSUE THREAD

**Issue by laurentribot** [ansible/ansible-modules-core#5409]
##### ISSUE TYPE
- Bug Report
##### COMPONENT NAME

include_role
##### ANSIBLE VERSION

```
ansible 2.2.0.0
  config file = /home/userdev/Documents/dlp-ansible/deploy_aqz/ansible.cfg
  configured module search path = ['library']
```

(branch stable-2.2)
##### CONFIGURATION

local roles and libraries location
##### OS / ENVIRONMENT

Master: Ubuntu 16.04.2
Managed: Rhel 6.6
##### SUMMARY

privileges are not pass to nested role
##### STEPS TO REPRODUCE

```
- hosts: all
  gather_facts: True

  tasks:
    - command: "whoami"

    - include_role:
        name: "role_test_a"
      become: "yes"
      become_user: "user2"
```

role_test_a/tasks/main.yml

```

---
- command: "whoami"

- include_role:
    name: "role_test_b"
```

role_test_b/tasks/main.yml

```

---
- command: "whoami"
```
##### EXPECTED RESULTS

```
PLAY [all] *********************************************************************

TASK [setup] *******************************************************************
ok: [host]

TASK [command] *****************************************************************
changed: [host] => {"changed": true, "cmd": ["whoami"], "delta": "0:00:00.002667", "end": "2016-10-27 09:37:19.915705", "rc": 0, "start": "2016-10-27 09:37:19.913038", "stderr": "", "stdout": "user1", "stdout_lines": ["user1"], "warnings": []}

TASK [role_test_a : command] ***************************************************
changed: [host] => {"changed": true, "cmd": ["whoami"], "delta": "0:00:00.004548", "end": "2016-10-27 09:37:20.550015", "rc": 0, "start": "2016-10-27 09:37:20.545467", "stderr": "", "stdout": "user2", "stdout_lines": ["user2"], "warnings": []}

TASK [role_test_b : command] ***************************************************
changed: [host ] => {"changed": true, "cmd": ["whoami"], "delta": "0:00:00.002869", "end": "2016-10-27 09:37:21.134721", "rc": 0, "start": "2016-10-27 09:37:21.131852", "stderr": "", "stdout": "user2", "stdout_lines": ["user2"], "warnings": []}

PLAY RECAP *********************************************************************
host : ok=7    changed=3    unreachable=0    failed=0 
```
##### ACTUAL RESULTS

```
PLAY [all] *********************************************************************

TASK [setup] *******************************************************************
ok: [host]

TASK [command] *****************************************************************
changed: [host] => {"changed": true, "cmd": ["whoami"], "delta": "0:00:00.002667", "end": "2016-10-27 09:37:19.915705", "rc": 0, "start": "2016-10-27 09:37:19.913038", "stderr": "", "stdout": "user1", "stdout_lines": ["user1"], "warnings": []}

TASK [role_test_a : command] ***************************************************
changed: [host] => {"changed": true, "cmd": ["whoami"], "delta": "0:00:00.004548", "end": "2016-10-27 09:37:20.550015", "rc": 0, "start": "2016-10-27 09:37:20.545467", "stderr": "", "stdout": "user2", "stdout_lines": ["user2"], "warnings": []}

TASK [role_test_b : command] ***************************************************
changed: [host ] => {"changed": true, "cmd": ["whoami"], "delta": "0:00:00.002869", "end": "2016-10-27 09:37:21.134721", "rc": 0, "start": "2016-10-27 09:37:21.131852", "stderr": "", "stdout": "user1", "stdout_lines": ["user1"], "warnings": []}

PLAY RECAP *********************************************************************
host : ok=7    changed=3    unreachable=0    failed=0 
```

**Comment by ansibot** [comment]
@ansible ping, this issue is waiting for your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@ansible, ping. This issue is waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by danielkza** [comment]
I'm also observing the same problem (Ansible 2.2.0 from PyPI, on Ubuntu 16.04).

**Comment by ansibot** [comment]
@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@laurentribot: Thanks for taking the time to open this issue, but it was filed under the wrong github repository. Please create a new issue at https://github.com/ansible/ansible-modules-ansible/issues
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by danielkza** [comment]
@ansibot That repository does not exist. Are you misbehaving?

**Comment by jctanner** [comment]
@danielkza yeah, that's a bug in the bot. It should have directed you to https://github.com/ansible/ansible/issues because all of the real code for include_role is there.
